dali/operators/reader/nvdecoder/imgproc.cu

```diff
@@ -41,6 +41,21 @@ __constant__ float ycbcr2rgb_mat[9] = {
   1.164383f * 255.0f,  2.017232f * 255.0f,  0.0f
 };
 
+
+// https://en.wikipedia.org/wiki/YUV#Y%E2%80%B2UV444_to_RGB888_conversion
+__constant__ float ycbcr2rgb_mat_norm_full_range[9] = {
```

Comment:
I guess we could use color space conversion functions - though it may be a bit out of scope.

Reply:
We have this strange float output path. I would move this rework to another PR.

Reply:
Done

Reply:
Reverted due to subtle discrepancies.